test/proposal_test.rb

```diff
@@ -24,156 +24,17 @@
 require "storage"
 require "y2storage"
 require_relative "support/proposal_examples"
```

Comment:
`proposal_examples` is not used here, I believe. Which makes me thing it should be called `proposal_scenario_examples`, BTW.